internal/operators/lso/ls_operator.go

```diff
@@ -0,0 +1,84 @@
+)
+
+type lsOperator struct {
+}
+
+// New LSOperator creates new instance of a Local Storage Operator installation plugin
+func NewLSOperator() *lsOperator {
+	return &lsOperator{}
+}
+
+// GetType reports type of an operator this Operator manages
+func (l *lsOperator) GetType() models.OperatorType {
+	return models.OperatorTypeLso
+}
+
+// GetDependencies provides a list of dependencies of the Operator
+func (l *lsOperator) GetDependencies() []models.OperatorType {
+	return make([]models.OperatorType, 0)
+}
+
+// GetClusterValidationID returns cluster validation ID for the Operator
+func (l *lsOperator) GetClusterValidationID() string {
+	return string(models.ClusterValidationIDLsoRequirementsSatisfied)
+}
+
+// GetHostValidationID returns host validation ID for the Operator
+func (l *lsOperator) GetHostValidationID() string {
+	return string(models.HostValidationIDLsoRequirementsSatisfied)
+}
+
+// ValidateCluster always return "valid" result
+func (l *lsOperator) ValidateCluster(ctx context.Context, cluster *common.Cluster) (api.ValidationResult, error) {
+	return api.ValidationResult{Status: api.Success, ValidationId: l.GetClusterValidationID(), Reasons: []string{}}, nil
+}
+
+// ValidateHost always return "valid" result
+func (l *lsOperator) ValidateHost(context.Context, *common.Cluster, *models.Host) (api.ValidationResult, error) {
+	return api.ValidationResult{Status: api.Success, ValidationId: l.GetHostValidationID(), Reasons: []string{}}, nil
+}
+
+// GetCPURequirementForWorker provides worker CPU requirements for the operator
+func (l *lsOperator) GetCPURequirementForWorker(context.Context, *common.Cluster) (int64, error) {
+	return 0, nil
+}
+
+// GetCPURequirementForMaster provides master CPU requirements for the operator
+func (l *lsOperator) GetCPURequirementForMaster(context.Context, *common.Cluster) (int64, error) {
+	return 0, nil
+}
+
+// GetMemoryRequirementForWorker provides worker memory requirements for the operator
+func (l *lsOperator) GetMemoryRequirementForWorker(context.Context, *common.Cluster) (int64, error) {
+	return 0, nil
+}
+
+// GetMemoryRequirementForMaster provides master memory requirements for the operator
+func (l *lsOperator) GetMemoryRequirementForMaster(context.Context, *common.Cluster) (int64, error) {
+	return 0, nil
+}
+
+// GenerateManifests generates manifests for the operator
+func (l *lsOperator) GenerateManifests(c *common.Cluster) (*api.Manifests, error) {
+	manifestFiles, err := Manifests(c.Cluster.OpenshiftVersion)
+	return &api.Manifests{Files: manifestFiles}, err
+
+}
+
+// GetDisksRequirementForMaster provides a number of disks required in a master
+func (l *lsOperator) GetDisksRequirementForMaster(context.Context, *common.Cluster) (int64, error) {
+	return 0, nil
+}
+
+// GetDisksRequirementForWorker provides a number of disks required in a worker
+func (l *lsOperator) GetDisksRequirementForWorker(context.Context, *common.Cluster) (int64, error) {
+	return 1, nil
```

Comment:
Could you please explain on why it is returning 1?

Reply:
Actually, that should probably be "2", as the validation requires the nodes to have at least one disk other than the boot disk.

Reply:
@nbalacha, what should it return? Should we allow it to return 0? Because the disk resource requirements will be taken care in cluster validations.

Reply:
Yes, we can return 0

Reply:
:+1: Done.